mpizzagalli/WoloxTraining
# Links

<h1> Summary
- Deleted test folder as told in comments of previous PR.
- Add feature for listing all articles in a table.
- Add links to navigate through the blog.

<h1> Test Plan
- Go to http://localhost:3000/articles and see the list of all the articles.
- Create a new article, and check that is included in the list of all the articles.
- Navigate through the blog.

<h1> Trello Card

https://trello.com/c/LVPfSJ2w/24-crear-blog-base

<h1> Screenshots

New view of the welcome index, with the link to the blog:
![home_small](https://cloud.githubusercontent.com/assets/7007384/9137487/47c4a3d8-3cf4-11e5-9a5d-c2e3a4a65c20.png)
View of the list of all the articles:
![articles_small](https://cloud.githubusercontent.com/assets/7007384/9137484/42bc7370-3cf4-11e5-94a6-31b8acff832d.png)
New view of one article, with the back link:
![article_small](https://cloud.githubusercontent.com/assets/7007384/9137486/44b3671a-3cf4-11e5-9fb4-b5eeff45ef18.png)
New view of the form for creating a new article, with the back link:
![new_article_small](https://cloud.githubusercontent.com/assets/7007384/9137488/48d3fe9a-3cf4-11e5-8ced-6ce74b475358.png)
